app/ViewStores.tsx

```diff
@@ -67,7 +67,7 @@ export function ViewStores(props: { onCreateClick: () => void }): ReactElement {
   });
   return (
     <Wrapper>
-      {displayedStores}
+      {storeList.length !== 0 ? displayedStores : "Loading"}
```

Comment:
this is not strictly true, if there are no stores at all we should not see `Loading` because that's misleading

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/24/commits/a7b94f148b9c3b11f9ca32380130ca0d7b2a479a